dev-resources/demo-schema.edn

```diff
@@ -2,7 +2,7 @@
  {:tick
   {:description "A subscription response."
    :fields {:time_ms {:type String
-                      :description "Time when tick is emittied (as string-ified long milliseconds since epoch)."
+                      :description "Time when tick is emitted (as string-field long milliseconds since epoch)."
```

Comment:
`ified` wasn't really a typo, I was trying to express that time was a numeric string, not a number.